### PR TITLE
Fixup new examples to use the latest API

### PR DESCRIPTION
**Pull request by garethr**
This also addresses the issue of from being a reserved work in python,
the list of packages is moved to fromPackages instead.

### CODE REVIEW

File: examples/api-demo-2b-issues-analytics.py [hunk 1]
```diff
@@ -41,232 +40,320 @@ def parse_command_line_args():
...
+                        curIsOnWatchlist,
+                        curDep.packageManager,
+                        curDep.fromPackages[0],
+                        tmpUpgradePath,
+                    ]
+                    self.fixFound = True
+
+    # Analysis worker
+    def analyzeRepo(self, issueset):
+        for v in issueset.issues.vulnerabilities:
```
Comment (jdewinne) [inline examples/api-demo-2b-issues-analytics.py]: I tried running the example, but it breaks at this line.
Traceback (most recent call last):
  File "examples/api-demo-2b-issues-analytics.py", line 358, in <module>
    masterlog.analyzeRepo(issueset)
  File "examples/api-demo-2b-issues-analytics.py", line 305, in analyzeRepo
    for v in issueset.issues.vulnerabilities:
AttributeError: 'IssueSetManager' object has no attribute 'issues'
```suggestion
        for v in issueset.all().issues.vulnerabilities:
```

File: snyk/managers.py [hunk 2]
```diff
@@ -296,6 +296,16 @@ def all(self) -> Any:
 
 
 class IssueSetManager(SingletonManager):
+    def _convert_reserved_words(self, data):
+        if "issues" in data:
+            if "vulnerabilities" in data["issues"]:
```
Comment (jdewinne) [inline snyk/managers.py]: Wouldn't this be nicer in a 1 liner?
```suggestion
            if "issues" in data and "vulnerabilities" in data["issues"]:
```

File: snyk/managers.py [hunk 3]
```diff
@@ -296,6 +296,16 @@ def all(self) -> Any:
 
 
 class IssueSetManager(SingletonManager):
+    def _convert_reserved_words(self, data):
+        if "issues" in data:
```
Comment (jdewinne) [inline snyk/managers.py]: This also need to be extended for `licenses` and not only vulns

Comment (garethr) [reply snyk/managers.py]: In master, licenses is still a dict. Should be easy to port in a follow up or the PR that changes licenses to an object?

File: snyk/managers.py (hunk 2 above)
Comment (garethr) [reply snyk/managers.py]: Thanks, fixed.

File: examples/api-demo-2b-issues-analytics.py (hunk 1 above)
Comment (garethr) [reply examples/api-demo-2b-issues-analytics.py]: Thanks, fixed.

**Comment by garethr** [comment]
@jdewinne thanks for the review, suggested fixes made. Ready for another review.

**Pull request merged**